devices.js

```diff
@@ -1142,9 +1142,9 @@ const devices = [
         model: 'SSM-U01',
         vendor: 'Xiaomi',
         description: 'Aqara single switch module T1 (with neutral)',
-        fromZigbee: [fz.on_off, fz.metering, fz.electrical_measurement, fz.device_temperature],
-        exposes: [e.switch(), e.energy(), e.power(), e.device_temperature()],
-        toZigbee: [tz.on_off],
+        fromZigbee: [fz.on_off, fz.metering, fz.electrical_measurement, fz.device_temperature, fz.aqara_switchtype, fz.aqara_power_outage_memory],
+        exposes: [e.switch(), e.energy(), e.power(), e.device_temperature(), exposes.enum('switchtype', exposes.access.ALL, ['toggle', 'momentary']), e.power_outage_memory()],
```

Comment:
`exposes.enum('switchtype', exposes.access.ALL, ['toggle', 'momentary'])` can be moved to `exposes.js`. Also add a description to it: `exposes.enum('switchtype', exposes.access.ALL, ['toggle', 'momentary']).withDescription("Control the behaviour of the switch")`

Reply:
Moved to `exposes.js`